README.md

```diff
@@ -0,0 +1,155 @@
+ 
+Just like with Twitter, a user should be able to take any actions (except sign-up), unless they are logged in. Once a user is logged in, they should be able to create, edit and delete their own tweets, as well as view all the tweets.
+
+There are controller tests to make sure that you 
+
+## Group Project Instructions
+
+*Instructions for how to work on a Group Project with Learn*
+
+### Some Hints on Working Together 
+
+Working on a software project with another person is not something to be taken lightly. While you are a fantastic coder solo, software development is a collaborative activity. Just like anything else, there is skill in collaborating on code. In the end, collaborating with another person boils down to three different styles:
+
+  - Pair - Pair the entire time working linearly together
+  - Pass - 1 person does 1 requirement and then the next person does the next one
+  - Parallel - work on different parts at the same time by agreeing on interfaces and stubs and meeting in the middle
+
+Remember! The goal at The Flatiron School is not to do, it is to *learn*. Make sure you have worked in all three styles of collaboration. We want you to learn how the different styles works, and make sure that together you and your partner understand every part of the code.
+
+## Instructions
+
+### File Structure
+
+```
+├── CONTRIBUTING.md
+├── Gemfile
+├── Gemfile.lock
+├── LICENSE.md
+├── README.md
+├── Rakefile
+├── app
+│   ├── controllers
+│   │   └── application_controller.rb
+│   ├── models
+│   │   ├── tweet.rb
+│   │   └── user.rb
+│   └── views
+│       ├── index.erb
+│       ├── layout.erb
+│       ├── tweets
+│       │   ├── create_tweet.erb
+│       │   ├── edit_tweet.erb
+│       │   ├── show_tweet.erb
+│       │   └── tweets.erb
+│       └── users
+│           ├── create_user.erb
+│           └── login.erb
+├── config
+│   └── environment.rb
+├── config.ru
+├── db
+│   ├── development.sqlite
+│   ├── migrate
+│   │   ├── 20151124191332_create_users.rb
+│   │   └── 20151124191334_create_tweets.rb
+│   ├── schema.rb
+│   └── test.sqlite
+└── spec
+    ├── controllers
+    │   └── application_controller_spec.rb
+    └── spec_helper.rb
+```
+
+### Gemfile and environment.rb
+
+This project is supported by Bundler and includes a `Gemfile`.
+
+Run bundle install before getting started on the project.
+
+As this project has quite a few files, an `environment.rb` is included that loads all the code in your project along with Bundler. You do not ever need to edit this file. When you see require_relative `../config/environment`, that is how your environment and code are loaded.
+
+### Models
+
+You'll need to create two models in `app/models`, one `User` model and one `Tweet`. Both classes should inherit from `ActiveRecord::Base`.
+
+### Migrations
+
+You'll need to create two migrations to create the users and the tweets table.
+
+Users should have a username, email, and password.
+
+Tweets should have content, and a user_id.
```

Comment:
Don't tell them about the user_id field, just say users have many tweets and have them figure it out